File: Anul 2/Semestrul 2/MPP/labs/Tema 4-proiect Server-Client Csharp/proiectSCcsharp/angajatModel/Angajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proiectSCcsharp.domain
{
    [Serializable]
    public class Angajat : Entity<long>
    {
        public string name { get; set; }
        public string username { get; set; }
        public string password { get; set; }

        public Angajat(string Name, string Username, string Password)
        {
            name = Name;
            username = Username;
            password = Password;
        }
        public override string ToString()
        {
            return name + " | " + username + " | " + password;
        }
    }
}

File: Anul 2/Semestrul 2/MPP/labs/Tema 5-proiect Server Csharp-Client Java/proiectServerClientJava/Domain/src/main/java/proiectServerClientJava/domain/Angajat.java
package proiectServerClientJava.domain;

import java.io.Serial;
import java.io.Serializable;

public class Angajat extends Entity<Long> implements Serializable {
    private String name;
    private String username;
    private String password;
    public Angajat(Long id,String name, String username, String password) {
        super(id);
        this.name = name;
        this.username = username;
        this.password = password;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
